metricflow/test/fixtures/semantic_manifest_yamls/simple_manifest/saved_queries.yaml

```diff
@@ -0,0 +1,12 @@
+---
+saved_query:
+  name: p0_booking
+  description: Booking-related metrics that are of the highest priority.
+  metrics:
+    - bookings
+    - instant_bookings
+  group_bys:
```

Comment:
@plypaul should this be group_by?

Reply:
Was there a spec change? This is following https://github.com/dbt-labs/metricflow/issues/765

Reply:
That's a typo in the spec. We use `group_by` everywhere else, so it should be `group_by` for saved queries.

We used to call them `group_bys` but it was horribly confusing for everybody so we switched to `group_by`.